firefox-ios/Shared/Strings.swift

```diff
@@ -7574,6 +7574,29 @@ extension String {
     }
 }
 
+// MARK: - Live Activity
+extension String {
+    public struct LiveActivity {
+        public struct Downloads {
+            public static let FileNameText = MZLocalizedString(
+                key: "LiveActivity.Downloads.FileNameText.v137",
+                tableName: "LiveActivity",
+                value: "Downloading \"%@\"",
+                comment: "Displayed during a download in Live Activity or Dynamic Island. The %@ placeholder represents the filename of the file being downloaded (e.g. Downloading \"MyFile.pdf\")")
```

Comment:
@Delphine do we need to clarify for translators that these slashes are escape sequences? I've never actually had to deal with that so I'm not sure what the L10N policy on this is. :)

Reply:
It's best to use smart quotes, and that will work